inspirehep/config.py

```diff
@@ -1188,3 +1188,9 @@ def _(x):
     "stat.OT": "Other",
     "stat.TH": "Other"
 }
+
+# Import local config file if it is present.
+try:
+    from inspirehep.config_local import *
```

Comment:
And where is the `config_local` file? In Puppet for example we are filling the `inspirehep.cfg` file that goes into `var/instance`. When would we use the `config_local` file?